solution/ui/regulations/stories/SupplementalContent.stories.js

```diff
@@ -13,7 +15,27 @@ const Template = (args, { argTypes }) => ({
 
 export const Basic = Template.bind({});
 Basic.args = {
-    "title": "43",
+    "api_url": "http://localhost:8000/v2/",
+    "title": "42",
     "part": "433",
-    "sections": ["100", "200", "300"]
+    "sections": ["100", "200", "300"],
+    "getSupplementalContentLegacy": () => Promise.resolve(supplementalContentResponse)
```

Comment:
not resolving with mock data if application is not running locally; all requests failing as they try to reach out to localhost:8000